### ISSUE TITLE
Keras word2vec is much more faster than tensorflow word2vec ?

### ISSUE THREAD

**Issue by muthiyanbhushan** [adventuresinML/adventures-in-ml-code#3]
Thanks for your work.
I had a question about word2vec.

I ran both of your codes of word2vec viz Keras_word2vec.py and tf_word2vec.py !! 

Keras word2vec with Tensorflow backend seems faster than Tensorflow word2vec. Ideally it should not. Keras is indirectly calling Tensorflow.
Tensorflow code took 1182 sec to run 15 iterations whereas Keras just took 796 seconds to run 15 iterations.

How is Keras faster than Tensorflow?

Can you please help me?

Thanks.

**Comment by Santosh-Gupta** [comment]
it looks like the tensorflow uses a different function to calculate loss compared to the keras version, so the two are not comparable.
